libbls.h

```diff
@@ -0,0 +1,30 @@
+/* libbls Header Version 0.1.0 */
+
+#include <stdlib.h>
+#include <stdbool.h>
+
```

Comment:
The header should contains other things. For example, it should contains the constants for the keys' size:

```c
const int SK_SIZE = 32;
const int SIG_SIZE = 48;
const int PK_SIZE = 96;
```

And also, a proper enum (a C enum, do not be confused with Rust enums, they cannot be ported yet in FFI easily):

```c
typedef enum {
  BLS_OK = 0,
  BLS_INVALID_BYTES = 1,
  BLS_VERIFICATION_FAILED = 2
} bls_sign_status;
```

(I just called `bls_sign_status` but call it whatever you want to)

Of course that it means you have to change the returned value of the methods below, from `int` to `bls_sign_status`, for example:
```c
bls_sign_status sign(const uint8_t *sk_ptr, 
		const uint8_t *pk_ptr, 
		const uint8_t *msg_ptr, 
		size_t msg_len,
		const uint8_t *sig_ptr);
```